test/unit/specs/components/StreamingMedia/createMediaEventManager.spec.js

```diff
@@ -130,6 +130,7 @@ describe("StreamingMedia::createMediaEventManager", () => {
     });
     expect(eventManager.sendEvent).toHaveBeenCalledWith(event, {
       mediaOptions,
+      edgeConfigOverrides: undefined,
```

Comment:
Instead of changing the test to have `edgeConfigOverrides: undefined`, we should continue the work of #1199 and modify [`src/core/createEventManager.js`](https://github.com/adobe/alloy/blob/main/src/components/StreamingMedia/createMediaEventManager.js#L80) to not include empty `edgeConfigOverrides` in the call to `sendEvent`

Reply:
It is using src/core/createEventManager.js down the line, which checks the `edgeConfigOverrides` and construct the request payload based on that.